README.md

```diff
@@ -351,6 +351,68 @@ app.addActions({votes: voteActions})
 
 </details>
 
+### `app.addListener(listener)`
```

Comment:
## Changes
New Documentation for `addListener` function.
The example is pretty much copy-paste from `addActions`